device/Microsoft.Azure.Devices.Client.NetStandard/Microsoft.Azure.Devices.Client.NetStandard.csproj

```diff
@@ -189,31 +198,12 @@
     <Compile Include="System.Web\Util\UriUtil.cs" />
     <Compile Include="System.Web\Util\Utf16StringValidator.cs" />
   </ItemGroup>
-  <ItemGroup>
```

Comment:
@jasmineymlo pointed out that we still need the loc generation for now.
Adding this back.